optaplanner-test/src/main/java/org/optaplanner/test/impl/score/stream/DefaultMultiConstraintVerification.java

```diff
@@ -51,7 +51,7 @@ public final DefaultMultiConstraintAssertion given(Object... facts) {
     public final DefaultMultiConstraintAssertion givenSolution(Solution_ solution) {
         try (InnerScoreDirector<Solution_> scoreDirector = scoreDirectorFactory.buildScoreDirector(true, true)) {
             scoreDirector.setWorkingSolution(Objects.requireNonNull(solution));
-            return new DefaultMultiConstraintAssertion<>(constraintProvider, scoreDirector.calculateScore(),
+            return new DefaultMultiConstraintAssertion(constraintProvider, scoreDirector.calculateScore(),
```

Comment:
Any reason we're using less generics and more raw types here?

Reply:
I didn't want to touch the public API, which I'd have to do to add the `<Score_>` type there.
I could do that, but again - MVP.

Reply:
Good point - I managed to limit adding the Score_ param to the impl classes only, but we probably want to make MultiConstraintAssertion Score_ sensitive? Different issue.